### PR TITLE
Rework bundles and GitHub release

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by lyarwood** [comment]
https://gist.github.com/maxmoehl/7fe32619f8503b5d29abfcc0e0f9e7df

File: .github/scripts/update-ssp.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    exit 1
+fi
+
+# Set git configs to sign the commit
+git config --global user.email "${RELEASE_FORK_USER}@redhat.com"
+git config --global user.name "common-instancetypes Release Automation"
+
+# Clone repo and use a token to allow pushing before creating a PR
+git clone https://github.com/kubevirt/ssp-operator && cd ssp-operator || exit
+git checkout origin/master -b update-common-instancetypes-"${COMMON_INSTANCETYPES_VERSION}"
```
Comment (0xFelix) [inline .github/scripts/update-ssp.sh:14]: Needs to be main

File: .github/scripts/update-ssp.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+if [ -z "${COMMON_INSTANCETYPES_VERSION}" ] || [ -z "${COMMON_INSTANCETYPES_RELEASE_URL}" ] || [ -z "${RELEASE_FORK_USER}" ] || [ -z "${GH_TOKEN}" ]; then
+    echo "${BASH_SOURCE[0]} expects the following env variables to be provided: COMMON_INSTANCETYPES_VERSION, COMMON_INSTANCETYPES_RELEASE_URL, RELEASE_FORK_USER and GH_TOKEN."
+    exit 1
+fi
+
+# Set git configs to sign the commit
+git config --global user.email "${RELEASE_FORK_USER}@redhat.com"
```
Comment (0xFelix) [inline .github/scripts/update-ssp.sh:9]: Hardcode domain part of email?

File: .github/scripts/update-ssp.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+cp ../common-clusterpreferences-bundle-"${COMMON_INSTANCETYPES_VERSION}".yaml data/common-instancetypes-bundle/
+
+git add data && git commit -sm "common-instancetypes: Update bundle to COMMON_INSTANCETYPES_VERSION ${COMMON_INSTANCETYPES_VERSION}"
+
+git remote add release https://"${RELEASE_FORK_USER}":"${GH_TOKEN}"@github.com/"${RELEASE_FORK_USER}"/ssp-operator
+git push -f -u release update-common-instancetypes-"${COMMON_INSTANCETYPES_VERSION}"
+
+# Create a new PR in the operator repo
+gh pr create --repo kubevirt/ssp-operator \
+    --base master \
```
Comment (0xFelix) [inline .github/scripts/update-ssp.sh:27]: Needs to be main

**Pull request closed without merging**